frontend/src/services/UserService.ts

```diff
@@ -16,17 +15,27 @@ export const registerUser = (user: IUpdateUserData) =>
     },
   }).then(response => response.json());
 
-// unused right now as Khoury auth is being used
-export const loginUser = (user: ILoginData) =>
+interface LoginUserData {
+  errors?: any;
+  user?: any;
+}
+
+interface LoginUserResponse {
+  status: number;
+  data: LoginUserData;
+}
+
+export const loginUser = (user: ILoginData): Promise<LoginUserResponse> =>
   fetch("/api/users/login", {
     method: "POST",
     body: JSON.stringify({ user: user }),
     headers: {
       "Content-Type": "application/json",
     },
-  }).then(response => response.json());
+  }).then(response => {
+    return { status: response.status, data: response.json() as LoginUserData };
```

Comment:
adding the status to the returned object since it'll be useful to differentiate between 401s and other things